merlin/datasets/ecommerce/aliccp/raw/schema.pbtxt

```diff
@@ -0,0 +1,294 @@
+    }
+  }
+}
+feature {
+  name: "user_intentions"
+  type: INT
+  int_domain {
+    name: "user_intentions"
+    max: 33786
+    is_categorical: true
+  }
+  annotation {
+    tag: "categorical"
+    tag: "user"
+    extra_metadata {
+      type_url: "type.googleapis.com/google.protobuf.Struct"
+      value: "\n:\n\010cat_path\022.\032,.//categories/unique.user_intentions.parquet\n\030\n\013start_index\022\t\021\000\000\000\000\000\000\000\000\nG\n\017embedding_sizes\0224*2\n\026\n\tdimension\022\t\021\000\000\000\000\000\000\200@\n\030\n\013cardinality\022\t\021\000\000\000\000@\177\340@\n\025\n\010max_size\022\t\021\000\000\000\000\000\000\000\000\n\017\n\tis_ragged\022\002 \000\n\034\n\017dtype_item_size\022\t\021\000\000\000\000\000\000@@\n\033\n\016freq_threshold\022\t\021\000\000\000\000\000\000\000\000\n\021\n\013num_buckets\022\002\010\000\n\r\n\007is_list\022\002 \000"
+    }
+  }
+}
+feature {
+  name: "user_brands"
+  type: INT
+  int_domain {
+    name: "user_brands"
+    max: 58015
+    is_categorical: true
+  }
+  annotation {
+    tag: "categorical"
+    tag: "user"
+    extra_metadata {
+      type_url: "type.googleapis.com/google.protobuf.Struct"
+      value: "\n\033\n\016freq_threshold\022\t\021\000\000\000\000\000\000\000\000\n\025\n\010max_size\022\t\021\000\000\000\000\000\000\000\000\n6\n\010cat_path\022*\032(.//categories/unique.user_brands.parquet\n\r\n\007is_list\022\002 \000\n\034\n\017dtype_item_size\022\t\021\000\000\000\000\000\000@@\n\017\n\tis_ragged\022\002 \000\n\030\n\013start_index\022\t\021\000\000\000\000\000\000\000\000\nG\n\017embedding_sizes\0224*2\n\026\n\tdimension\022\t\021\000\000\000\000\000\000\200@\n\030\n\013cardinality\022\t\021\000\000\000\000\340S\354@\n\021\n\013num_buckets\022\002\010\000"
+    }
+  }
+}
+feature {
+  name: "user_categories"
+  type: INT
+  int_domain {
+    name: "user_categories"
+    max: 6086
+    is_categorical: true
+  }
+  annotation {
+    tag: "categorical"
+    tag: "user"
+    extra_metadata {
+      type_url: "type.googleapis.com/google.protobuf.Struct"
+      value: "\n\034\n\017dtype_item_size\022\t\021\000\000\000\000\000\000@@\nG\n\017embedding_sizes\0224*2\n\026\n\tdimension\022\t\021\000\000\000\000\000`j@\n\030\n\013cardinality\022\t\021\000\000\000\000\000\306\267@\n\025\n\010max_size\022\t\021\000\000\000\000\000\000\000\000\n\030\n\013start_index\022\t\021\000\000\000\000\000\000\000\000\n\021\n\013num_buckets\022\002\010\000\n\r\n\007is_list\022\002 \000\n:\n\010cat_path\022.\032,.//categories/unique.user_categories.parquet\n\017\n\tis_ragged\022\002 \000\n\033\n\016freq_threshold\022\t\021\000\000\000\000\000\000\000\000"
+    }
+  }
+}
+feature {
+  name: "click"
+  type: INT
+  annotation {
```

Comment:
We need the tag BINARY_CLASSIFICATION  [here](https://github.com/NVIDIA-Merlin/models/blob/833b84c1575ce76e583450272f6ae1ee9fef4b34/merlin/models/data/synthetic.py#L234) to generate the corresponding vector with zeros and ones. 

I understand we need to remove the tags to avoid  tagging the target encoding variables we got from nvt workflow  with `Target` tag. So maybe we can remove the `TARGET` tag and keep the binary classification tag?